HOWTO

```diff
@@ -2861,11 +2861,6 @@ Threads, processes and job synchronization
 	``flow=8`` and another job has ``flow=-1``, then there will be a roughly 1:8
 	ratio in how much one runs vs the other.
 
-.. option:: flow_watermark=int
-
-	The maximum value that the absolute value of the flow counter is allowed to
-	reach before the job must wait for a lower value of the counter.
-
```

Comment:
I'm not sure about throwing away the `flow_watermark` behaviour as that will just break old jobs (but I'm all for the examples being converted to the new way as you did below). It would be better to deprecate the old option and remove it entirely in the future.